tests/test_api/test_validation.py

```diff
@@ -582,3 +583,62 @@ def test_check_opera_rtc_s1_date_max_configurable(monkeypatch):
         validation.check_opera_rtc_s1_date(
             {'job_parameters': {'granules': ['S1_000000_IW1_20250506T000000_VV_0000-BURST']}}, None
         )
+
+
+def test_check_aria_s1_gunw_dates(monkeypatch):
```

Comment:
I'm not seeing `monkeypatch` used in this test:
```suggestion
def test_check_aria_s1_gunw_dates():
```